### DISCUSSION TITLE
Seek in gameplay will not clear the judgement line in `HitErrorMeter`

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#25713] [Q&A] [answered]
I noticed while writing a function that [Clear()](https://github.com/ppy/osu/blob/a553387c3d77d7d4e695242939fe69c41190c63a/osu.Game/Screens/Play/HUD/HitErrorMeters/HitErrorMeter.cs#L69) in `HitErrorMeter` will never call.

this is because `gameplayClockContainer` will always null
![image](https://github.com/ppy/osu/assets/34775378/43eb3222-b543-4ebd-bd8d-8e1e6bffb41c)
https://github.com/ppy/osu/blob/a553387c3d77d7d4e695242939fe69c41190c63a/osu.Game/Screens/Play/HUD/HitErrorMeters/HitErrorMeter.cs#L45

I'm a little confused, I feel like there's something wrong with it.

**Comment by bdach** [comment]
Likely broke when the component was changed to be a skinnable.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/25747
